products/nextjs.md

```diff
@@ -19,8 +19,13 @@ auto:
   methods:
   -   npm: next
 
-# eol(x) = MAX(releaseDate(x+1), latestReleaseDate(x))
```

Comment:
relates to https://github.com/endoflife-date/endoflife.date/pull/4240

open to update it (maybe like releaseDate(x+1) + 1yr)?